Source/diablo.cpp

```diff
@@ -2094,4 +2094,4 @@ void diablo_color_cyc_logic()
 	}
 }
 
-DEVILUTION_END_NAMESPACE
+}

```

Comment:
it'd be good to have a terminating comment for namespaces, `} // namespace devilution`

clang-tidy will do this automatically https://clang.llvm.org/extra/clang-tidy/checks/llvm-namespace-comment.html

Reply:
Sure

Reply:
> it'd be good to have a terminating comment for namespaces

Can you elaborate why @glebm ? This seems really unnecessary to me, especially because it _only_ applies to namespaces for some weird reason. I don't appreciate inconsistent rules like that at all :S

Reply:
Well, I'm not really sure why, I'm just cargo-culting what I'm used to here.
No strong opinion.

It is applied automatically though, so once @AJenbo adjusts .clang-tidy rules it won't be something you'll have to thing about